yt_dlp/extractor/ard.py

```diff
@@ -455,6 +463,7 @@ def _real_extract(self, url):
             'subtitles': subtitles,
             'is_live': is_live,
             'age_limit': age_limit,
+            'chapters': chapters,
```

Comment:
We can simplify this using a single `traverse_obj` call:
```suggestion
            **traverse_obj(media_data, {
                'chapters': ('pluginData', 'jumpmarks@all', 'chapterArray', ..., {
                    'start_time': ('chapterTime', {int_or_none}),
                    'title': ('chapterTitle', {str}),
                }),
            }),
```